build.sbt

```diff
@@ -3,4 +3,18 @@ version := "0.1"
 scalaVersion := "2.12.4"
 
 // https://mvnrepository.com/artifact/uk.co.jhc/sqlest
-libraryDependencies += "uk.co.jhc" %% "sqlest" % "0.8.10"
+libraryDependencies ++= Seq(
+  "com.typesafe" % "config" % "1.3.0",
+  "uk.co.jhc" %% "sqlest" % "0.8.8",
```

Comment:
Has reverted the version slightly, I'd find the latest version from the Github page,